enhancements/sig-policy/134-standalone-hub-templates/README.md

```diff
@@ -0,0 +1,185 @@
+# Standalone Hub Templates
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+To support Open Cluster Management (OCM) hub templates without the policy framework, this feature provides an additional
+`governance-standalone-hub-templating` addon which generates a user per cluster on the hub to be leveraged by the
+Configuration Policy controller for hub template resolution. The OCM hub administrator will be responsible for granting
+`list` and `watch` permissions to the resources needed by the hub templates. A sample `ConfigurationPolicy` can be a way
+to automate this for a cluster set.
+
+## Motivation
+
+When an Open Cluster Management (OCM) policy user deploys their policies with external tools such as Argo CD, the hub
+templating functionality is lost. This is often a critical feature for large scale deployments and prevents users from
+choosing an external tool if they require an alternative to the OCM policy framework.
+
+### Goals
+
+1. Support hub templates for `ConfigurationPolicy` and `OperatorPolicy` when deployed through an external tool such as
+   Argo CD.
+1. Hub templates should resolve when referenced objects are updated on the hub.
+1. Policies can still be evaluated if the managed cluster is disconnected from the hub.
+1. Permissions to the hub are per managed cluster.
+1. Provide a hub only `ConfigurationPolicy` to provide permissions to cluster sets.
+1. It must work without the rest of the policy framework on the managed cluster.
+
+### Non-Goals
+
+1. `CertificatePolicy` and Gatekeeper constraints will not be directly supported. It can be wrapped in a
+   `ConfigurationPolicy` if hub templates are needed for those.
+1. Updating a `ConfigurationPolicy` definition with hub templates while disconnected from the hub is not explicitly
+   supported. This will almost always work based on the proposed design as long as there aren't new hub template API
+   queries, but if the update happened while the hub and Configuration Policy controller were offline, it will not work.
```

Comment:
I’m confused. The goal says, 'Policies can still be evaluated if the managed cluster is disconnected from the hub,' but it also states, 'If the hub and Configuration Policy controller are offline, it won’t work.' I don’t understand the difference.

Reply:
To me "Policies can still be evaluated if the managed cluster is disconnected from the hub." doesn't make sense to me. Can we add more to this sentense

Reply:
I added more details. Does this make sense now?

Reply:
Perfect THanks!!